### PR TITLE
#155820624: Launch DetailActivity from MainActivity

### PR DESCRIPTION
**Pull request by omokolataiwo**
### What does this PR do?
Lunches User Details Activity from Main Activity

### Description of Task to be completed?
- [x] Add DetailActivity
- [x] Launch DetailActivity from MainActivity

### How should this be manually tested?
- Clone the repository
- Open project in Android Studio
- Click `run app`

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
[Finishes: #155820624]

### Screenshots (if appropriate)
fast

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -12,10 +14,12 @@
         <activity android:name=".MainActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
-
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".DetailActivity"
+            android:parentActivityName=".MainActivity"
+            android:label="Profile"></activity>
```
Comment (chykehyman) [inline app/src/main/AndroidManifest.xml]: Since the label for this activity is generated dynamically, you can as well get rid of this hardcoded `Profile` label here.

File: app/src/main/java/com/andela/omokolataiwo/levelup/ListAdapter.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        final String profileImage = itemList.getProfilePicture();
+
+        holder.usernameTextView.setText(username);
+        Picasso.get().load(profileImage).into(holder.profilePictureImageView);
+
+        holder.itemView.setOnClickListener( new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Intent intent = new Intent(context, DetailActivity.class);
+                intent.putExtra("USER_NAME", username);
```
Comment (chykehyman) [inline app/src/main/java/com/andela/omokolataiwo/levelup/ListAdapter.java]: As part of standards, strings should be written in lowercase. 
For instance, it's either you declare a string as a direct `key` of an `intent` value or you indirectly assign a string to a constant variable.  E.g `public static final String USER_NAME = "username";` then you use it like this => `intent.putExtra(USERNAME, username);`

File: app/build.gradle [hunk 3]
```diff
@@ -20,11 +20,12 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation 'com.android.support:appcompat-v7:27.1.1'
+    implementation 'com.android.support:appcompat-v7:27.1.0'
```
Comment (chykehyman) [inline app/build.gradle:23]: Why did you change your dependency version codes from `.1` to `.0`

File: app/src/main/res/layout/activity_user_detail.xml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                tools:layout_editor_absoluteX="21dp"
+                tools:layout_editor_absoluteY="388dp" />
+
+            <TextView
+                android:id="@+id/tv_github_address"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="45dp"
+                android:fontFamily="casual"
+                android:text="http://github.com/username"
```
Comment (chykehyman) [inline app/src/main/res/layout/activity_user_detail.xml]: Always extract your string variables to the `string resource directory`
Or...
since the text in this TextView will be dynamically replaced at runtime, u can instead replace `android:text` with `tools:text`

File: app/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Top Github User</string>
+    <string name="app_name">Github Android Devs.</string>
```
Comment (chykehyman) [inline app/src/main/res/values/strings.xml]: This application is based on `Java` not `Android` devs

**Review by chykehyman — COMMENTED** [review]
Please act accordingly  to my reviews. When done, you can merge.

In general, nice job.

File: app/build.gradle (hunk 3 above)
Comment (omokolataiwo) [reply app/build.gradle:23]: I had to change the dependency version code from .1 to .0 because the Picasso **com.squareup.picasso:picasso:2.71828** internally depended on **v7:27.1.0** and all dependency must use the same dependency version code.

Comment (chykehyman) [reply app/build.gradle:23]: @omokolataiwo 
Hmm... how strange!
